.github/workflows/package.yml

```diff
@@ -43,14 +43,15 @@ jobs:
   package-linux:
     name: Linux
     # Oldest supported runner, for wide glibc compat
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
     steps:
       - uses: actions/checkout@v4
         with:
           lfs: true
       - name: Install dependencies
         run: sudo apt update && sudo apt-get -y install libasound2-dev libvulkan-dev libfontconfig-dev
-      # No prebuilt shaderc, since the official binaries don't to be compatible with Ubuntu 20.04.
+      # No prebuilt shaderc, since the official binaries don't seem to be compatible with Ubuntu 20.04,
```

Comment:
Is this something we should address in this PR? I'm wondering if it's possible to use a prebuilt shaderc with Ubuntu 22.04.

Reply:
It'd speed up the builds a lot, but no need for it to happen in this PR.